rental_base/views/menu_view.xml

```diff
@@ -84,78 +84,90 @@
     <menuitem
         name="Customers"
         id="menu_top_customer"
+        groups="sales_team.group_sale_salesman"
```

Comment:
We should change this to base.group user. No need to protect the top menu.
A stock user for example would not see the outgoing deliveries if this is only visible to sales people.